script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -1,37 +1,37 @@
 [sha:174548517d159082577a6222d1f566275c277d40]{
 -- Page 10018, Offset 25709.
 -- こつこつと時計の針は進んでいく。
-The hands of the clock advances and ticks away.
+The hands on the clock advance.
```

Comment:
The clock's hands

Reply:
I disagree, "the hands on the clock" is kinda a set expression

Reply:
np then